.travis.yml

```diff
@@ -4,9 +4,10 @@
 
 language: node_js
 node_js:
+  - 'node'
   - '10'
-  - '9'
   - '8'
+  - '6'
```

Comment:
Same, please remove.

Reply:
After adding `engines` to package.json, yes.